keras_core/layers/preprocessing/hashed_crossing_test.py

```diff
@@ -36,6 +37,12 @@ def test_basics(self):
         )
 
     def test_correctness(self):
```

Comment:
We should have shared tests for all backends plus a small set of tests that rely on string inputs and are therefore TF only (there are no string tensors in other backends).

Reply:
@fchollet sure let me make these changes.

Reply:
@fchollet done.